src/backend/libc/process/types.rs

```diff
@@ -381,6 +381,8 @@ pub type RawUid = c::uid_t;
 /// A CPU identifier as a raw integer.
 #[cfg(any(target_os = "android", target_os = "linux"))]
 pub type RawCpuid = u32;
+#[cfg(not(target_os = "wasi"))]
+pub type RawId = c::id_t;
```

Comment:
This is annoying, but `procctl` requires an `id_t` argument and `id_t` is not `pid_t` on FreeBSD.

(I don't know why it uses `id_t`, since the value must be either a pid or a pgid, and `pid_t` and `pgid_t` are both `i32`, while `id_t` is `i64`.)